turbo.json

```diff
@@ -1,5 +1,6 @@
 {
   "$schema": "https://turbo.build/schema.json",
+  "dangerouslyDisablePackageManagerCheck": true,
```

Comment:
i didn't see mention as to why we removed `packageManager` from the `package.json` - is it just maintenance upkeep cost?

Reply:
Check the nodejs-website channel. There are many issues caused by that field + different people use different versions + npm commands were failing and CI was failing.